contributing/documentation/overview.rst

```diff
@@ -35,48 +57,82 @@ Let's imagine that you want to improve the installation chapter of the Symfony
 book. In order to make your changes, follow these steps:
 
 **Step 1.** Go to the official Symfony documentation repository located at
-`github.com/symfony/symfony-docs`_ and `fork the repository`_ to your personal
-account. This is only needed the first time you contribute to Symfony.
+`github.com/symfony/symfony-docs`_ and click on the **Fork** button to `fork the
+repository`_ to your personal account. This is only needed the first time you
+contribute to Symfony.
 
-**Step 2.** **Clone** the forked repository to your local machine (this
-example uses the ``projects/symfony-docs/`` directory to store the documentation;
-change this value accordingly):
+**Step 2.** **Clone** the forked repository to your local machine (this example
+uses the ``projects/symfony-docs/`` directory to store the documentation; change
+this value accordingly):
 
 .. code-block:: bash
 
     $ cd projects/
     $ git clone git://github.com/<YOUR GITHUB USERNAME>/symfony-docs.git
 
-**Step 3.** Switch to the **oldest maintained branch** before making any change.
-Nowadays this is the ``2.3`` branch:
+**Step 3.** Add the original Symfony docs repository as a "Git remote" executing
+this command:
 
 .. code-block:: bash
 
     $ cd symfony-docs/
-    $ git checkout 2.3
+    $ git remote add upstream https://github.com/symfony/symfony-docs.git
+
+If things went right, you'll see the following when listing the "remotes" of
+your project:
+
+.. code-block:: bash
+
+    $ git remote -v
+    origin  https://github.com/<YOUR GITHUB USERNAME>/symfony-docs.git (fetch)
+    origin  https://github.com/<YOUR GITHUB USERNAME>/symfony-docs.git (push)
```

Comment:
For your own repository, their shouldn't be HTTPS URLs (we cloned using Git over SSH before).